platformio.ini

```diff
@@ -293,30 +293,35 @@ upload_protocol   = serial
 
 #
 # BigTree SKR Mini V1.1 / SKR mini E3 / SKR E3 DIP (STM32F103RCT6 ARM Cortex-M3)
+# Newer boards ship with a 512K chip labeled as RCT6.
```

Comment:
not really... i got mine when its out... its a RCT6 and have 512K and 64K ram.... newer ones seems to have RET6 printed on them

Reply:
I didn't add that. It was just part of a header I moved.

Reply:
I think it varies depending on the board, the latest E3 mini v1.2 boards have RCT6, just got mine 2 weeks ago. But the DIP boards are shipping with RET6 now I think

Reply:
That is interesting. I have both E3 DIP and E3 Mini 1.0 boards I received in early September and both have RET6 processors.

I'm guessing they have specified RCT6 for these boards, but allow RET6 as a drop-in replacement. If that is the case the part used would vary depending on which part is cheaper or more available that day.